ui/ducks/send/send.test.js

```diff
@@ -768,7 +768,7 @@ describe('Send Slice', () => {
       it('should not error with an invalid address error when user input is not a valid hex string', () => {
         const tokenAssetTypeState = {
           ...INITIAL_SEND_STATE_FOR_EXISTING_DRAFT,
-          recipientInput: '0xValidateError',
+          recipientInput: '0xAAAA6BF26964aF9D7eEd9e03E53415D37aA96045',
         };
```

Comment:
This change probably needs to be removed for tests to pass

Reply:
@danjm, Just to confirm - is this test not needed and currently not present in the develop branch and should it be removed?

Reply:
ok done.